pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
     "prtpy",
     "pydot",
     "dataclasses-json",
-    "sdx-datamodel @ git+https://github.com/atlanticwave-sdx/datamodel@2.0.6.rc4",
+    "sdx-datamodel @ git+https://github.com/atlanticwave-sdx/datamodel@main",
```

Comment:
we can change this to a new release of the datamodel. Just made it from `main` because I didnt know the release process there.

Reply:
You might want to refer to a commit hash or tag rather than `main`, in order to prevent breakage, because `main` is a moving target.  The release process is super lightweight -- push a release tag to datamodel, refer to that tag from pce, push a release tag to pce, and eventually use the new pce release tag in sdx-controller.

An alternative would be getting CI to publish pce and datamodel packages somewhere, but PyPI.org would be the wrong place for it (because our libraries are not general purpose), and GitHub Packages registry does not support Python.  

I think the current workflow is adequate. We can look into alternatives if/when we need to.

Reply:
Turns out that referring to `datamodel@main` has broken pce's main branch, as mentioned above, because both datamodel APIs and assumptions that pce makes about topology representation has changed.  I have filed an issue: https://github.com/atlanticwave-sdx/pce/issues/200.

https://github.com/atlanticwave-sdx/pce/actions/runs/10271724419 is the CI run.

Reply:
Hi @sajith you are right. I should have acted more carefully on this case. I've added a comment on Issue #200  to comment about this error. Please let me know if you are okay on us moving this discussion over Issue #200! If that is so, please mark this discussion as resolved (I can do it if you prefer)